opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/RuntimeEnvironment.java

```diff
@@ -1426,30 +1426,28 @@ public void signalTorefreshSearcherManagers(List<String> subFiles, String host)
 
     /**
      * Generate a TreeMap of projects with corresponding repository information.
-     *
+     * <p>
      * Project with some repository information is considered as a repository
      * otherwise it is just a simple project.
      */
-    private void generateProjectRepositoriesMap() throws IOException {
-        synchronized (repository_map) {
-            repository_map.clear();
-            for (RepositoryInfo r : getRepositories()) {
-                Project proj;
-                String repoPath;
-                try {
-                    repoPath = PathUtils.getPathRelativeToSourceRoot(
-                            new File(r.getDirectoryName()), 0);
-                } catch (ForbiddenSymlinkException e) {
-                    LOGGER.log(Level.FINER, e.getMessage());
-                    continue;
-                }
+    public void generateProjectRepositoriesMap() throws IOException {
```

Comment:
does this really need to be public ? (esp. now that the `repository_map` locking is removed)

Reply:
no, it does not